api/src/main/java/org/pmiops/workbench/db/model/User.java

```diff
@@ -170,6 +171,18 @@ public void setAuthorities(Set<Authority> newAuthorities) {
     this.authorities = newAuthorities;
   }
 
+  @ElementCollection(fetch = FetchType.LAZY)
+  @CollectionTable(name = "page_visit", joinColumns=@JoinColumn(name="user_id"))
+  @OneToMany(cascade = CascadeType.ALL, orphanRemoval = true, fetch = FetchType.EAGER, mappedBy = "user")
```

Comment:
Is there a conflict between the two fetch types here? Lazy on line 174 and eager here on 176.

Reply:
Does not seem to be, and fails if I set the second one to LAZY, I think this goes into the issue @calbach spoke about earlier around these settings.

Reply:
I'm surprised this would still be needed. I do worry that this line means we always fetch page visits on every user fetch.

I will patch this now and see if I can figure out anything.

Reply:
updatePageVisits was reading `getPageVisits()` without using the new dao method you defined, this should fix:

https://github.com/all-of-us/workbench/compare/us/rw-1022...ch/try-lazy-visits?expand=1